.github/workflows/nuget.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Nuget
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Have you considered using https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release so that it will get pushed automatically?

I do like manual so we can test but less is more 😉

Reply:
I think manual is fine while we test, we can switch it later once we are more familiar with how it works.